### PR TITLE
Adding Bakhshali Sqrt Algorithm

### PR DESCRIPTION
**Pull request by archuser555**
This Is A Technique To Calculate Square Root, I Write It In Rust, And Yeah, It's Indian Old Way But Work's

### CODE REVIEW

File: src/math/bakhshali_sqrt.rs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#[allow(path_statements, unused_must_use)]
```
Comment (merelymyself) [inline src/math/bakhshali_sqrt.rs:1]: are these necessary?

Comment (archuser555) [reply src/math/bakhshali_sqrt.rs:1]: Kinda Of, Deleting Them Cause Some Warnings

Comment (archuser555) [reply src/math/bakhshali_sqrt.rs:1]: After Check I Found That They Actually Do Nothing :-)

File: src/math/bakhshali_sqrt.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    pub fn test_bakhshali_sqrt() {
+        // let's try get the sqrt of 10
+        let a: f32 = bakhshali_sqrt(9.0);
+        assert_eq!(a, 3.0); //Sqrt Of 9 Is 3
+    }
```
Comment (siriak) [inline src/math/bakhshali_sqrt.rs:43]: Please add more tests with 0, negative numbers, 0 to 1 numbers, non-perfect-square integer numbers

**Review by siriak — CHANGES_REQUESTED** [review]

File: src/math/bakhshali_sqrt.rs (hunk 2 above)
Comment (archuser555) [reply src/math/bakhshali_sqrt.rs:43]: Ok, Gonna Add Some!

Comment (archuser555) [reply src/math/bakhshali_sqrt.rs]: Done :-)

**Review by siriak — COMMENTED** [review]
The code looks good, but fix the build check please

**Comment by zim0369** [comment]
If OP is busy then I could make a PR in case this one is going to be abandoned. Do I?

**Comment by archuser555** [comment]
idk what's the prob :-)
ok do it! but mention me!
@zim0369
